app/importers/californica_importer.rb

```diff
@@ -44,7 +44,7 @@ def import
   def finalize_import
     parser.order_child_works
     parser.reindex_collections
-    parser.build_iiif_manifests
+    # parser.build_iiif_manifests
```

Comment:
If we disable via a comment, should we also include an explanation of why it is commented?